pom.xml

```
@@ -64,7 +64,7 @@
     <properties>
         <revision>2.42</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.273-rc30689.09147672b85b</jenkins.version>
+        <jenkins.version>2.176.4</jenkins.version>
```

can we use a supported one at least, most of the others were recently bumped to 2.222.4

```suggestion
        <jenkins.version>2.222.4</jenkins.version>
```